CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.0.1]
```

Comment:
Add link to 1.0.1 and 1.0.0 at the bottom of this page?

Reply:
Good catch, done. I wonder if we could extend our formatting suite slightly to do this automatically.